packages/metro/src/DeltaBundler/__tests__/traverseDependencies-integration-test.js

```diff
@@ -3094,6 +3151,64 @@ describe('traverseDependencies', function() {
         expect(deps).toMatchSnapshot();
       });
     });
+
+    it('should work with scoped providesModuleNodeModules (haste, win32)', async () => {
+      var root = 'C:\\root';
+      setMockFileSystem({
+        root: {
+          'index.js': `
+            /**
+             * @providesModule index
+             */
+             require('shouldWork');
+             require('Bar');
+          `,
+          node_modules: {
+            'react-haste': {
+              'package.json': JSON.stringify({
+                name: 'react-haste',
+                main: 'main.js',
+              }),
+              'main.js': `
+                /**
+                 * @providesModule shouldWork
+                 */
+                require('Bar');
+              `,
+            },
+            '@org': {
+              module: {
+                'package.json': JSON.stringify({
+                  name: '@org/module',
+                  main: 'main.js',
+                }),
+                'main.js': '// Blank',
+                'Bar.js': `
+                  /**
+                   * @providesModule Bar
+                   */
+                   require('shouldWork');
+                `,
+              },
+            },
+          },
+        },
+      });
+
+      const opts = {
+        ...defaults,
+        providesModuleNodeModules: [...defaults.providesModuleNodeModules, '@org/module'],
+        watchFolders: [root],
+      };
+      await processDgraph(opts, async dgraph => {
+        const deps = await getOrderedDependenciesAsJSON(
+          dgraph,
+          'C:\\root\\index.js',
+        );
+        console.error(deps)
```

Comment:
nit: can you remove the `console.error()`?

Reply:
Oops, forgot that debug line there! Fixed it up.